pints/_optimisers/__init__.py

```diff
@@ -753,7 +809,8 @@ class Optimisation(OptimisationController):
     """ Deprecated alias for :class:`OptimisationController`. """
 
     def __init__(
-            self, function, x0, sigma0=None, boundaries=None, method=None):
+            self, function, x0, sigma0=None, boundaries=None, method=None,
```

Comment:
Again think method is better off staying the last argument

Reply:
Done